redbot/cogs/audio/apis/interface.py

```diff
@@ -149,19 +149,19 @@ async def run_tasks(self, ctx: Optional[commands.Context] = None, message_id=Non
         async with self._lock:
             if lock_id in self._tasks:
                 if IS_DEBUG:
-                    log.debug(f"Running database writes for {lock_id} ({lock_author})")
+                    log.debug("Running database writes for %d (%s)", lock_id, lock_author)
                 try:
                     tasks = self._tasks[lock_id]
                     tasks = [self.route_tasks(a, tasks[a]) for a in tasks]
                     await asyncio.gather(*tasks, return_exceptions=False)
                     del self._tasks[lock_id]
                 except Exception as exc:
                     debug_exc_log(
-                        log, exc, f"Failed database writes for {lock_id} ({lock_author})"
+                        log, exc, "Running database writes for %d (%s)", lock_id, lock_author
```

Comment:
I think you didn't mean to change this?
```suggestion
                        log, exc, "Failed database writes for %d (%s)", lock_id, lock_author
```